### ISSUE TITLE
Deleted Important Notes

### ISSUE THREAD

**Issue by ShepLT1** [BoostIO/Boostnote#3715]
# Current behavior
Notes from 6/28 & 7/1 are absent

# Expected behavior
The notes should be here

# Steps to reproduce
Notes were missing after opening app on 7/6/21

1. 
2. 
3. 

# Environment

- Boostnote version: <0.16.1>
- OS version and name:

**Comment by Komediruzecki** [comment]
Hi ShepLT1,

can you please give more info of the issue, can you describe your storages, type of storage (cloud, local space, legacy etc.)?

Also, this is the github repository for old legacy boost note, did you meant to open it for the new boost note next?

Can you provide some screenshot about the application and how did you lost your notes? Did you have it synced up internally (legacy cloud sync or new cloud sync) or externally?

**Comment by ShepLT1** [comment]
Yes sorry, this is for boost note next. I am using local space for storage. I was under the impression that the app auto-saved on closure, but I just noticed the 'Save As' feature under the file tab so I'll use that. I haven't synced anything yet. Any screenshot would just be a note document with a section of its notes missing. I can take a screenshot if necessary, but would rather not.

**Comment by ShepLT1** [comment]
Also I just upgraded to version 0.20.1, but that didn't solve the issue

**Comment by Komediruzecki** [comment]
Hi again,

IF you are really using local storage, then the sync is not available (with cloud servers). But local storage stores all notes on disk, the notes are auto-saved but the timeout is a bit long (around few seconds), so if you just close the app after entering something, it might not have time to save last few seconds.

The save as feature in menu is for saving it to external file (markdown, PDF, HTML) that is exporting the note.
So if you are really losing your notes on file system storage, we should inspect this further, because auto-save should be on, always.

**Comment by bettydi** [comment]
After an update today I am no longer able to access my notes. These were local storage - can you tell me where I should look to see if I can recover them? Are they stored in a local database on disk? And if a database is used, which one? Thx.
